### ISSUE TITLE
replace Combine Future with async/await if any left

### ISSUE THREAD

**Issue by tomholub** [FlowCrypt/flowcrypt-ios#1058]
@tomholub I think we should use `async/await` here instead of Combine for code consistency across project, and for better threads handling with `@MainActor`, can be done in another issue

_Originally posted by @sosnovsky in https://github.com/FlowCrypt/flowcrypt-ios/pull/1055#discussion_r754064992_

**Comment by tomholub** [comment]
<img width="744" alt="image" src="https://user-images.githubusercontent.com/6306961/142852382-8ddb9fdb-b5d8-4d60-ab29-ad72f3940952.png">

**Comment by tomholub** [comment]
If after #1031 any are still left, we should remove them in this issue
